app/src/main/java/com/ciandt/dragonfly/example/features/modelselection/ModelSelectionActivity.kt

```diff
@@ -28,6 +29,8 @@ class ModelSelectionActivity : BaseActivity(), ModelSelectionContract.View {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_model_selection)
 
+        startService(RemoteModelService.create(this))
```

Comment:
Method just returns `Intent(context, RemoteModelService::class.java)`
Maybe the name should be createIntent().

What do you think?